doc/release-notes/rl-2511.section.md

```diff
@@ -110,6 +110,8 @@
 
 - [private-gpt](https://github.com/zylon-ai/private-gpt) service has been removed by lack of maintenance upstream.
 
+- `lxde` scope has been remove and lxde has been moved to `pkgs/by-name`.
```

Comment:
These are notes for users, not maintainers. `by-name` is an implementation detail.
```suggestion
- `lxde` scope has been removed, and its packages have been moved the top-level
```